src/sentry/receivers/savedsearch.py

```diff
@@ -3,30 +3,7 @@
 from django.db.models.signals import post_save
 
 from sentry.models import Project, SavedSearch
-
-DEFAULT_SAVED_SEARCHES = [
```

Comment:
This doesn't seem to have been added to the appropriate model file.

Reply:
Must have gotten lost in the splitting of diffs :( Will sort it out soon